test/adapter/json_api/linked_test.rb

```diff
@@ -153,11 +153,11 @@ def test_include_multiple_posts_and_linked
             serializer = BioSerializer.new @bio1
             adapter = ActiveModel::Serializer::Adapter::JsonApi.new(
               serializer,
-              include: ['author', 'author.posts']
+              include: [author: [:posts]]
```

Comment:
should we still have tests for the string equivalent of all these? just to be thorough?